src/build-time-render/BuildTimeRender.ts

```diff
@@ -124,9 +141,19 @@ export default class BuildTimeRender {
 			blockScripts.forEach((blockScript, i) => {
 				html = html.replace(
 					'</body>',
-					`<script type="text/javascript" src="${scriptPrefix}${blockScript}" async="true"></script></body>`
+					`<script type="text/javascript" src="${scriptPrefix}${blockScript}"></script></body>`
```

Comment:
Removed `async="true"` to increase the priority of script so the browser downloads it right away.